pkg/mimir/modules.go

```diff
@@ -361,7 +361,7 @@ func (t *Mimir) initDistributorService() (serv services.Service, err error) {
 	// ruler's dependency)
 	canJoinDistributorsRing := t.Cfg.isAnyModuleEnabled(Distributor, Write, All)
 
-	t.Cfg.Distributor.PreferStreamingChunks = t.Cfg.Querier.PreferStreamingChunks
+	t.Cfg.Distributor.PreferStreamingChunks = t.Cfg.Querier.PreferStreamingChunksFromIngesters
```

Comment:
Should we rename the distributor config field to `PreferStreamingChunksFromIngesters` too?